clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -6,14 +6,14 @@ Copyright  :  (C) 2013-2016, University of Twente,
 License    :  BSD2 (see the file LICENSE)
 Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
 
-BlockRAM primitives
+Block RAM primitives
 
 = Using RAMs #usingrams#
 
 We will show a rather elaborate example on how you can, and why you might want
-to use 'blockRam's. We will build a \"small\" CPU+Memory+Program ROM where we
-will slowly evolve to using blockRams. Note that the code is /not/ meant as a
-de-facto standard on how to do CPU design in Clash.
+to use 'blockRam's. We will build a \"small\" CPU + Memory + Program ROM where
+we will slowly evolve to using 'blockRam's. Note that the code is /not/ meant as
```

Comment:
I think I would distinguish between "block RAMs" the concept and `blockRam` the function, i.e. either write

> to using block RAMs.

or

> to using 'blockRam'.

If we do this, a function would never need to be pluralized, which feels slightly odd to me